static-sqlite-driver/build.gradle.kts

```diff
@@ -0,0 +1,189 @@
+import java.io.File
+import java.io.FileInputStream
+import com.powersync.plugins.sonatype.setupGithubRepository
+import de.undercouch.gradle.tasks.download.Download
+import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
+import org.jetbrains.kotlin.gradle.utils.NativeCompilerDownloader
+import org.jetbrains.kotlin.konan.properties.Properties
+import org.jetbrains.kotlin.konan.target.HostManager
+import org.jetbrains.kotlin.konan.target.KonanTarget
+import org.jetbrains.kotlin.konan.target.PlatformManager
+
+plugins {
+    alias(libs.plugins.kotlinMultiplatform)
+    alias(libs.plugins.downloadPlugin)
+    id("com.powersync.plugins.sonatype")
+}
+
+val sqliteVersion = "3450200"
+val sqliteReleaseYear = "2024"
```

Comment:
When adding this originally, I've copied the version from the previous `:core` build definition.

Any reason for not updating `sqlite-jdbc` and this to `3.49.1`?

Reply:
I could not find any reason not to update this. I originally used the same JDBC version as the core. I've updated all the versions now to `3.49.1`. Tests pass on all the platforms including the Swift SDK.